components/Navbar.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from "react";
+
+const Navbar: FC = () => {
+  return (
+    <div className="flex items-center justify-between py-10">
+      <h1 className="text-[32px] font-bold">Earcandy</h1>
+
+      <ul className="flex items-center gap-16 text-xl font-bold">
```

Comment:
make these links

Reply:
okay

Reply:
> add that sidebar as well, use `menu` that sagar created or if you have already created that sidebar, and is closer to design that sagar's then use yours in explore page as well, removing sagar's

Okay for now i am adding mine sidebar, we can remove it later if sagar's one look better okay?